<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Importer</class>
 <widget class="QMainWindow" name="Importer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFModpackImporter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="11" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Number Mods</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btn_browse_modpack">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="3">
     <widget class="QLineEdit" name="txt_modloader">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="3">
     <widget class="QLineEdit" name="txt_modpack_name">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Modloader</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Modpack Information</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Modpack Zip (Download form CurseForge)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="3">
     <widget class="QLineEdit" name="txt_mc_version">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1" colspan="3">
     <widget class="QLineEdit" name="txt_nummods">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Modpack Name</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="4">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>This tool will use Firefox or Chrome to automate downloading of the modpack mods. You will see many tabs opened and downloads happening automatically. Do not close the browser or this tool will fail.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLineEdit" name="txt_modpack_zip">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="4">
     <widget class="QPushButton" name="btn_generate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Download Mods and Generate Zip</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Minecraft Version</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Download Using</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QComboBox" name="cbox_browser">
      <item>
       <property name="text">
        <string>chrome</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>firefox</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_instructions"/>
    <addaction name="act_about"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="act_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="act_instructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
